### PR TITLE
feat(wedlocks) - nft email confirmation template

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description
Change email confirmation template with more details, tx URL and open sea URL are optional, so in that case the value will be print only if passed as parameter

- Add `opensea.tokenUrl` and `isTestNetwork` parameter is networks

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: wedlocks/src/templates/keyMined.js [hunk 1]
```diff
@@ -1,12 +1,28 @@
+const links = (params) => {
+  const hasTxUrl = params?.txUrl?.length > 0
```
Comment (julien51) [inline wedlocks/src/templates/keyMined.js:2]: I don't think txUrl has been defined

Comment (kalidiagne) [reply wedlocks/src/templates/keyMined.js:2]: added to parameters as optional

File: locksmith/src/utils/openSea.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  tokenId,
+  network,
+}: {
+  lockAddress: string
+  tokenId: string
+  network: any
+}): string | undefined => {
+  const isTestNetwork = [4, 31337, 80001].includes(network)
+
+  const baseUrl = isTestNetwork
```
Comment (julien51) [inline locksmith/src/utils/openSea.ts:14]: Can you check the network package and put that logic there please do it's reusable?

Thank you!

**Review by julien51 — CHANGES_REQUESTED** [review]

**Comment by julien51** [comment]
Also, now that I think about it the OpenSea URL should also be in the network packages (like we do for the block explorer!)

**Review by julien51 — APPROVED** [review]

File: locksmith/src/operations/wedlocksOperations.ts [hunk 3]
```diff
@@ -87,10 +86,14 @@ export const notifyNewKeysToWedlocks = async (
  * @param key
  */
 export const notifyNewKeyToWedlocks = async (key: any, network?: number) => {
+  const lockAddress = key.lock.address
+  const ownerAddress = key.owner.address
+  const tokenId = key?.keyId
```
Comment (julien51) [inline locksmith/src/operations/wedlocksOperations.ts:91]: Not sure we need `?` here, or we need it everywhere above as well

Comment (kalidiagne) [reply locksmith/src/operations/wedlocksOperations.ts:103]: `keyId` is not always passed as parameter, i update the `key: any` type definition to: 

```
 key: {
    lock: {
      address: string
      name?: string
    }
    owner: {
      address: string
    }
    keyId?: string
  },
  ```

File: locksmith/src/utils/openSea.ts (hunk 2 above)
Comment (kalidiagne) [reply locksmith/src/utils/openSea.ts]: added!

**Pull request merged**